CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Example for reservoir modeling using PINO and FNO
```

Comment:
Remove this line

Reply:
Yes, I removed - realized I did an incorrect merge conflict fix :)